src/olympia/activity/models.py

```diff
@@ -419,6 +419,13 @@ def f(self, *args, **kw):
         # SafeFormatter escapes everything so this is safe.
         return markupsafe.Markup(self.formatter.format(*args, **kw))
 
+    @classmethod
+    def transformer_hide_user_from_developer(cls, logs):
```

Comment:
I was hoping to do the actual anonymization in here, rather than just making the `user` `None`, but it didn't seem happy with me trying to change `log.user.username` in here. I am guessing we can only change things that exist on the `log` instance itself? Maybe there is a different way of doing it in here?